core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -1601,4 +1601,43 @@ private void validateDetailQueryBatchSize() {
       }
     }
   }
+
+  public boolean isDistributedPruningEnabled(String dbName, String tableName) {
```

Comment:
for all public method newly added, please write comment

Reply:
done